Bottom: 7ec7338f84ad24394a991922d814d58558b38bec
Top:    45053a2706db05cf3799df452c84f43f90fccacb
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 23:26:50 +0000

Refresh of play-with-quad-edge-data

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 28fb0fe..6463437 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -395,7 +395,6 @@ object3d_from_board_outline (void)
           if (offset_in_ct == ct_npoints)
             {
               start_of_ct = i;
-              printf ("start_of_ct = %i\n", start_of_ct);
               offset_in_ct = 0;
               ct = ct->next;
               ct_npoints = get_contour_npoints (ct);
